distributed/utils.py

```diff
@@ -817,7 +817,7 @@ def format_bytes(n):
 if PY2:
     def nbytes(frame):
         """ Number of bytes of a frame or memoryview """
-        if type(frame) in (bytes, buffer):
+        if type(frame) in (bytes, buffer, memoryview):
```

Comment:
See https://github.com/dask/distributed/issues/1159#issuecomment-308138962 for a correct implementation.